test/test_transformations.py

```diff
@@ -186,16 +188,23 @@ def test_drop_false(self):
 
 class DropShortTestCase(unittest.TestCase):
 
-    def _init_transformer(self, num_stations=None, keep_fakes=True):
-        self.transformer = DropShort(num_stations=num_stations, keep_fakes=keep_fakes)
+    def _init_transformer(self, num_stations=None, keep_filtered=True):
```

Comment:
methods that are required for every test should be called in the `setUpClass` or `setUp` method.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

Reply:
changed init_data to setUp, _init_transformer not, because it has arguments